src/NHapi.Base/Model/AbstractGroup.cs

```diff
@@ -527,7 +522,7 @@ private IStructure TryToInstantiateStructure(Type c, string name)
                         var argClasses = new[] { typeof(IGroup), typeof(IModelClassFactory) };
                         var argObjects = new object[] { this, myFactory };
                         var con = c.GetConstructor(argClasses);
-                        o = con.Invoke(argObjects);
+                        o = con?.Invoke(argObjects);
```

Comment:
doesn't this change the behaviour (or is that intended)? just compiling in my head, but previously, if `con` was null, it would throw, be caught in the next catch block and `o` would be set using `Activator.CreateInstance`. With this change, it would skip that and throw in the next block?

Reply:
You are correct, will undo this change.

Reply:
changed to `!.`.